README.md

```diff
@@ -15,30 +14,7 @@
    +  Create documents
    +  Delete documents, users
    +  Update documents, users
-
-## Endpoints
```

Comment:
Is there any reason you removed this, i think it's important because how do i know what routes are available and the HTTP method to use.